bucket/xournalpp.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.0.15",
+    "description": "A handwriting notetaking software with PDF annotation support",
+    "homepage": "https://github.com/xournalpp/xournalpp",
+    "license": "GPL-2.0",
```

Comment:
```suggestion
    "license": "GPL-2.0-only",
```

Reply:
Actually it's GPL-2.0-or-later. I'll update it as "GPL-2.0-or-later"